### ISSUE TITLE
1.4.2 version is not compatible with v3-sdk

### ISSUE THREAD

**Issue by ChaituVR** [Uniswap/v3-periphery#313]
```js
Error: Cannot find module '@uniswap/v3-periphery/artifacts/contracts/interfaces/IMulticall.sol/IMulticall.json'
[2022-09-19 18:35:23] Require stack:
[2022-09-19 18:35:23] - /workspace/node_modules/@uniswap/v3-sdk/dist/v3-sdk.cjs.development.js
[2022-09-19 18:35:23] - /workspace/node_modules/@uniswap/v3-sdk/dist/index.js
```

This happens when we tried to update `1.4.1` version to `1.4.2` version

**Comment by muhammadsamir11** [comment]
Did you find a solution?

**Comment by jakubdonovan** [comment]
> Did you find a solution?

I would also like to know. The artifacts folder is not there.

**Comment by ChaituVR** [comment]
Tried different ways but ended up using yarn resolutions to use `1.4.1` version, hope they fix it in next version update

**Comment by jdubpark** [comment]
`artifacts` folder is missing in `1.4.2` for some reason, for now the solution is downgrading to `1.4.1` to continue using the artifacts.

FYI, this works for me:
```json
{
    "@uniswap/sdk": "^3.0.3",
    "@uniswap/sdk-core": "^3.0.1",
    "@uniswap/smart-order-router": "^2.9.2",
    "@uniswap/v3-core": "^1.0.1",
    "@uniswap/v3-periphery": "^1.4.1",
    "@uniswap/v3-sdk": "^3.9.0",
}
```

command: `npm i @uniswap/v3-periphery@1.4.1`

**Comment by bigchiano** [comment]
> `artifacts` folder is missing in `1.4.2` for some reason, for now the solution is downgrading to `1.4.1` to continue using the artifacts.
> 
> FYI, this works for me:
> 
> ```json
> {
>     "@uniswap/sdk": "^3.0.3",
>     "@uniswap/sdk-core": "^3.0.1",
>     "@uniswap/smart-order-router": "^2.9.2",
>     "@uniswap/v3-core": "^1.0.1",
>     "@uniswap/v3-periphery": "^1.4.1",
>     "@uniswap/v3-sdk": "^3.9.0",
> }
> ```
> 
> command: `npm i @uniswap/v3-periphery@1.4.1`

Thank you, this solved the issue for me.

**Comment by dade-ca** [comment]
This workaround worked for me:

```json
  "resolutions": {
    "@uniswap/v3-periphery": "1.4.1"
  }
```
in `package.json`

**Comment by roigecode** [comment]
This also worked for me!

"dependencies": {
    "@uniswap/v3-periphery": "**1.4.1**",
  }

**Comment by hensha256** [comment]
fixed in 1.4.3, sorry this took so long...
